dev-docs/publisher-api-reference.md

```diff
@@ -1179,8 +1180,11 @@ If a custom adServerTargeting function can return an empty value, this boolean f
 
 ### pbjs.offEvent(event, handler, id)
 
+### pbjs.getEvents() ⇒ `Array`
+
 The methods `onEvent` and `offEvent` are provided for you to register
-a callback to handle a Prebid.js event.
+a callback to handle a Prebid.js event. While the method `getEvents` returns a copy of
+all emitted events.
```

Comment:
Can we change this to: The `getEvents` method returns a copy of all emitted events.